src/v2/Components/FlashBanner/EmailConfirmationLinkExpired.tsx

```diff
@@ -0,0 +1,97 @@
+import React from "react"
+import { commitMutation, graphql } from "react-relay"
+import { Button, Sans } from "@artsy/palette"
+import { EmailConfirmationLinkExpiredMutationResponse } from "v2/__generated__/EmailConfirmationLinkExpiredMutation.graphql"
+import {
+  AnalyticsSchema as Schema,
+  useSystemContext,
+  useTracking,
+} from "v2/Artsy"
+
+import createLogger from "v2/Utils/logger"
+
+const logger = createLogger("EmailConfirmationLinkExpired")
```

Comment:
Looking around at other use cases for `createLogger` we generally include the path